examples/kafka-streams-examples/README.md

```diff
@@ -0,0 +1,230 @@
+# Confluent's Kafka Streams Examples
```

Comment:
Yeah, initially we decided that all the docs should live on docs.confluent.io. In retrospect, this has not worked out well because everyone does what you did (look on github). Your solution sounds very reasonable to me. 

We should publish a TOC for docker docs which points to docs.confluent.io on the github README (sometime soon :) ). That would make discovering docs a lot easier. cc @samjhecht